src/cleanvision/issue_managers/duplicate_issue_manager.py

```diff
@@ -89,14 +125,45 @@ def find_issues(
             issue_type: {} for issue_type in to_compute
         }
 
-        for path in tqdm(filepaths):
-            image = Image.open(path)
-            for issue_type in to_compute:
-                hash = self._get_hash(image, self.params[issue_type])
-                if hash in issue_type_hash_mapping[issue_type]:
-                    issue_type_hash_mapping[issue_type][hash].append(path)
-                else:
-                    issue_type_hash_mapping[issue_type][hash] = [path]
+        if n_jobs is None:
+            if psutil_exists:
+                n_jobs = psutil.cpu_count(logical=False)  # physical cores
+            if not n_jobs:
+                # either psutil does not exist
+                # or psutil can return None when physical cores cannot be determined
+                # switch to logical cores
+                n_jobs = multiprocessing.cpu_count()
+
+        if n_jobs == 1:
```

Comment:
There's some repetition of code in this way of implementing. How I would do it is, create a base method that populates `result` dict, then create a wrapper around this base method for using in `multiprocessing.Pool()`
Base method would look like
```suggestion
        def compute_hash(path, to_compute, params):
            # logic
            return result{"path": path, "issue_type0": hash0, "issue_type1": hash1  }
```
Wrapper would look like
```suggestion
def compute_hash_wrapper(args):
    path = args["path"]
    to_compute = args["to_compute"]
    params = args["params"]
    return compute_hash(path, to_compute, params)
```